### ISSUE TITLE
Recolor rating Icon

### ISSUE THREAD

**Issue by hamburglar2160** [jellyfin/jellyfin-web#1230]
...needs an update so that it's more obvious the rating comes from IMDB and not Mother Russia. 
edit: it's not the imdb rating, but let's recolor it

The current star is hardwired in.
current:
![image](https://user-images.githubusercontent.com/61879508/81936714-a56f0f80-95c0-11ea-83aa-222d772fa15a.png)
preferred:
![image](https://user-images.githubusercontent.com/61879508/81936593-87a1aa80-95c0-11ea-8a49-4fe70bfd90d5.png)

**Comment by MrTimscampi** [comment]
It's not an IMDB rating UNLESS your main provider is OMDB, where CommunityRating will be the IMDB rating.

For other providers, it will have a different meaning (For TMDB, it's the rating on TMDB, for example).

**Comment by TheGitPanda** [comment]
You can customise the colour of the star using custom CSS: 
```
.mediaInfoTimerIcon, .starIcon {
    color: #e8cd10; // (yellow)
}
```
Dashboard > General > Custom CSS

**Comment by ferferga** [comment]
Fixed in #1206
